xsuite/src/world/lsworld.test.ts

```diff
@@ -387,6 +387,82 @@ test.concurrent("LSWorld.setPreviousBlockInfo", async () => {
   assertVs(returnData, [e.U64(10), e.U64(20), e.U64(30), e.U64(40)]);
 });
 
+test.concurrent("LSWorld.advanceTimestamp", async () => {
+  using world = await LSWorld.start();
+  await world.advanceTimestamp(10);
+  const networkStatus = await world.proxy.getNetworkStatus(0);
+  expect(networkStatus).toEqual({
```

Comment:
1. Let's rather use `.toMatchObject` and only test blockTimestamp, round, epoch, nonce. We could even decide to only check the timestamp honestly.

2. We can't really check if the advanceTimestamp was successful by just checking the network status after, we also need to check the network status before.

3. Let's use `.setCurrentBlockInfo` to set an initial timestamp. Then let's check the timestamp. Then advance. Then check the timestamp.